sdk/test/IntegrationTests/Tests/General.cs

```diff
@@ -680,6 +680,7 @@ public void TestManualClockCorrection()
         // By default it only tests a small subset of services.
         [TestMethod]
         [TestCategory("General")]
+        [Ignore]

```

Comment:
This test always fails at least once before succeeding, and it's currently commented out in the NetStandard project. @peterrsongg is working on a SEP for clock skew, this test should be revisited after that.

Reply:
Yes, updated the reason for skipping the tests in https://github.com/aws/aws-sdk-net/pull/3556/commits/042cf27049f1eb325c11853e0b26ff20b15dc9b0